gatsby-node.js

```diff
@@ -27,14 +27,20 @@ exports.createPages = async ({ graphql, actions }) => {
       const pagePath =
         subPath === '' ? `${endpoint.name}/` : `${endpoint.name}/${subPath}/`;
 
-      createPage({
+      const pageProps = {
         path: pagePath,
         component: path.resolve(`src/subpages/${file}`),
         isPermanent: true,
         context: {
           endpoint,
         },
-      });
+      };
+
+      if (file === 'list.tsx' || file === 'search.tsx') {
+        createPage(pageProps);
+        return;
+      }
+      createPage({ ...pageProps, matchPath: pagePath + ':id' });
```

Comment:
search page can't found.

`account`, `block`, `transaction` subpage use `:id` path.

Reply:
This mechanism requires server-side rendering and without this links will work only at client-side, i.e., only available through links, not direct access to URLs.  Here's the repro:

```bash
./build.sh
cd _site/
npx http-server 8989
```

And then try to open <http://127.0.0.1:8989/9c-beta/block/fe41ed19f20ae643828d9fea514085a6069ce0a7c2752e19e3939a2d89000000> on the web browser:

![404 Not Found](https://user-images.githubusercontent.com/12431/89866908-4cc50800-dbeb-11ea-81e6-ae4000542cb5.png)

The page won't be loaded, since the server could not find the file like *9c-beta/block/fe41ed19f20ae643828d9fea514085a6069ce0a7c2752e19e3939a2d89000000*.

Reply:
Ok I'll revert change.